launch.go

```diff
@@ -216,7 +216,7 @@ func convertToArray(i interface{}) (array []int) {
 	}
 }
 
-func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, storeURL, uiURL, shellBin string, buildTimeout int, buildToken string) error {
+func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, storeURL, uiURL, shellBin string, buildTimeout int, buildToken string, cacheStrategy string, pipelineCacheDir string, jobCacheDir string, eventCacheDir string) error {
```

Comment:
you can do the short form `cacheStrategy, pipelineCacheDir, jobCacheDir, eventCacheDir string` for these new args are of type `string`

Reply:
done